components/MintPage/MintV2/SuccessModal.tsx

```diff
@@ -32,7 +32,7 @@ const SuccessModal = ({ isModalVisible, toggleIsVisible, cre8orNumber }) => {
             dark:text-black text-white
             leading-[103.3%]"
         >
-          {`#${cre8orNumber}`} Cre8or DNA Secured.
+          {`${quantity}`} Cre8or{`${quantity > 0 ? "s" : ""}`} DNA Minted.
```

Comment:
will this ever popup if quantity is less than zero? My understanding is that the default mint quantity is 1 so this will always be > 0.

Reply:
fixed 👍